CC/Sounder/config.cc

```diff
@@ -222,6 +246,17 @@ Config::Config(const std::string& jsonfile, const std::string& directory)
         cl_rxgain_vec_.at(1).assign(
             jClRxgainB_vec.begin(), jClRxgainB_vec.end());
 
+        if (std::any_of(cl_txgain_vec_.at(0).begin(),
+                cl_txgain_vec_.at(0).end(), compare(81))) {
```

Comment:
Name 81....   static const kMaxTxGain = 81?
You could also construct compare outside of the any_of call and pass in the callable object.